tests/Integration/Listing/ScriptFinderTest.php

```diff
@@ -54,8 +74,31 @@ public function test_script_finder_prefixes_script_names_with_namespace_if_prese
 
     public function test_script_finder_throws_exception_if_path_is_not_valid()
     {
-        $finder = new ScriptFinder([new ScriptPath(__DIR__ . '/_scripts_not_valid_directory')]);
+        $finder = new ScriptFinder(
+            [new ScriptPath(__DIR__ . '/_scripts_not_valid_directory')],
+            $this->descriptionReaderMock
+        );
+        
         $this->expectException(ScriptPathNotValidException::class);
         $finder->getAllScripts();
     }
+
+    public function test_script_finder_adds_description_to_script()
+    {
+        $finder = new ScriptFinder(
+            [new ScriptPath(__DIR__ . '/_scripts')],
+            new DescriptionReader()
+        );
+
+        $script = $finder->findScriptByName('description');
+        $this->assertSame('My description', $script->getDescription());
+    }
+
+    /**
+     * @before
+     */
+    public function createEmptyDescriptionReaderMock()
```

Comment:
please remove. And if you need a method like that I would prefer a setUp method on top of the test file